yaml/pois.yaml

```diff
@@ -172,6 +172,14 @@ filters:
     min_zoom: 17
     output:
       kind: {col: tags->zoo}
+  - filter: {waterway: waterfall}
+    min_zoom: CASE WHEN to_number(tags->'height', '9999.99') < 10 THEN 15
```

Comment:
+1 for porting the python logic to SQL so everything is more legible (both for waterfalls and peaks).